src/engraving/property/propertyvalue.cpp

```diff
@@ -237,7 +237,8 @@ PropertyValue PropertyValue::fromQVariant(const QVariant& v, P_TYPE type)
     case P_TYPE::DYNAMIC_TYPE:     return PropertyValue(DynamicType(v.toInt()));
     case P_TYPE::DYNAMIC_RANGE:    return PropertyValue(DynamicRange(v.toInt()));
     case P_TYPE::DYNAMIC_SPEED:    return PropertyValue(DynamicSpeed(v.toInt()));
-    case P_TYPE::HOOK_TYPE:    return PropertyValue(HookType(v.toInt()));
+    case P_TYPE::HOOK_TYPE:        return PropertyValue(HookType(v.toInt()));
+    case P_TYPE::KEYMODE:          return PropertyValue(KeyMode(v.toInt()));
```

Comment:
You also need to add to the `toQVariant` method

Reply:
Like up after line 172 add `case P_TYPE::KEYMODE:        return static_cast<int>(value<KeyMode>());`, right?

Reply:
right!

Reply:
Done